src/attribute-editor/__tests__/attribute-editor.test.tsx

```diff
@@ -208,6 +209,19 @@ describe('Attribute Editor', () => {
       const wrapper = renderAttributeEditor({ additionalInfo: 'test' });
       expect(wrapper.findAdditionalInfo()!.getElement()).toHaveTextContent('test');
     });
+
+    test('is connected to add button with aria-describedby', () => {
+      const wrapper = renderAttributeEditor({ ...defaultProps, additionalInfo: 'Test Info' });
+      const buttonElement = wrapper.findAddButton().getElement();
+      const info = wrapper.findAdditionalInfo()?.getElement();
+      expect(buttonElement).toHaveAttribute('aria-describedby', info?.id);
+    });
+
+    test('is part of an ARIA live region', () => {
+      const wrapper = renderAttributeEditor({ ...defaultProps, additionalInfo: 'Test Info' });
+      const liveRegion = wrapper.find(`.${liveRegionStyles.root}`)?.getElement();
+      waitFor(() => expect(liveRegion).toHaveTextContent('Test Info'));
```

Comment:
Missing await before `waitFor`?

Reply:
Oops, my bad! This assertion was wrong in the first place, anyway. I won't find the text here if I'm using `visible={true}`. Changed the assertion to just check for the existence of a live region.